config/lava/iec-security/iec-security.jinja2

```diff
@@ -0,0 +1,13 @@
+- test:
+    name: {{ plan }}
+    description: "cyber security standard tests such as IEC-62443-4-2"
+    timeout:
+      minutes: 20
+    definitions:
+    - repository: https://github.com/kernelci/iec-security.git
```

Comment:
Ah, this repository doesn't exist yet.  We need to create a fork and then it should be all good to go.

Reply:
Oh it does exist: https://github.com/kernelci/iec-security
But it's not being updated like the test-definitions one.  So that's why there's no `kernelci.org` branch or `staging.kernelci.org` branch; that's the next thing to enable.